Google.PowerShell.Tests/Common/TestActiveUserCredential.cs

```diff
@@ -0,0 +1,138 @@
+        /// </summary>
+        [Test]
+        public void TestRefreshTokenAsync()
+        {
+            ActiveUserCredential activeUserCred = new ActiveUserCredential();
+            ActiveUserToken activeUserToken = TokenProperty.GetValue(null, null) as ActiveUserToken;
+
+            bool refreshed = activeUserCred.RefreshTokenAsync(CancelToken).Result;
+
+            Assert.IsTrue(refreshed, "RefreshTokenAsync should return true.");
+
+            ActiveUserToken refreshedActiveUserToken = TokenProperty.GetValue(null, null) as ActiveUserToken;
+
+            Assert.IsTrue(
+                !Equals(activeUserToken.AccessToken, refreshedActiveUserToken.AccessToken),
+                "Refreshed ActiveUserToken should have a different access token.");
+        }
+
+        /// <summary>
+        /// This test checks that GetAccessTokenForRequestAsync works.
+        /// </summary>
+        [Test]
+        public void TestGetAccessTokenForRequestAsync()
+        {
+            ActiveUserCredential activeUserCred = new ActiveUserCredential();
+            ActiveUserToken activeUserToken = TokenProperty.GetValue(null, null) as ActiveUserToken;
+
+            string accessToken = activeUserCred.GetAccessTokenForRequestAsync(null, CancelToken).Result;
+
+            // The access token returned by GetAccessTokenForRequestAsync should come from token.
+            Assert.IsTrue(
+                Equals(activeUserToken.AccessToken, accessToken),
+                "GetAccessTokenForRefreshAsync returns the wrong access token.");
+
+            // The next call to GetAccessTokenForRequestAsync should returns the same token.
+            accessToken = activeUserCred.GetAccessTokenForRequestAsync(null, CancelToken).Result;
+            Assert.IsTrue(
+                Equals(activeUserToken.AccessToken, accessToken),
+                "GetAccessTokenForRefreshAsync returns the wrong access token.");
+        }
+
+        /// <summary>
+        /// This test checks that GetAccessTokenForRequestAsync returns a new token
+        /// when the old one expired due to time.
+        /// </summary>
+        [Test]
+        public void TestGetAccessTokenExpiredByTime()
+        {
+            ActiveUserCredential activeUserCred = new ActiveUserCredential();
+            ActiveUserToken activeUserToken = TokenProperty.GetValue(null, null) as ActiveUserToken;
+
+            // Make the token expired by time.
+            activeUserToken.Issued = DateTime.Now.AddSeconds(-activeUserToken.ExpiresInSeconds.Value);
+            Assert.IsTrue(activeUserToken.IsExpired(SystemClock.Default), "ActiveUserToken should be expired");
+
+            string newAccessToken = activeUserCred.GetAccessTokenForRequestAsync(null, CancelToken).Result;
+            Assert.IsFalse(
+                Equals(activeUserToken.AccessToken, newAccessToken),
+                "GetAccessTokenForRefreshAsync should returns a new access token when the old one expired due to time."
+                );
```

Comment:
When spilling function parameters across multiple lines, put the final `);` on the same line as the last parameter.